### PR TITLE
Updating package maintainers

### PR DESCRIPTION
**Pull request by quarkytale**
Apologies if I missed or miscredited anyone 🙏

Added maintainers and authors in alphabetical order, depending on contributors of each package (or rather everyone should be added in every package?).

Notes:
- Need Louise's and Joan's current/preferred email
- Added Rob in buoy_gazebo to credit for `mbari_wec`

### CODE REVIEW

File: buoy_description/package.xml [hunk 1]
```diff
@@ -4,9 +4,16 @@
...
+
+  <maintainer email="hamilton@mbari.org">Andrew Hamilton</maintainer>
+  <maintainer email="dharini@openrobotics.org">Dharini Dutia</maintainer>
+  <maintainer email="mabel@openrobotics.org">Mabel Zhang</maintainer>
+  <maintainer email="anderson@mbari.org">Michael Anderson</maintainer>
+
   <license>Apache 2.0</license>
+
   <author>Louise Poubel</author>
+  <author email="michael@openrobotics.org">Michael Caroll</author>
```
Comment (mabelzhang) [inline buoy_description/package.xml:16]:
```suggestion
  <author email="michael@openrobotics.org">Michael Carroll</author>
```

File: buoy_gazebo/package.xml [hunk 2]
```diff
@@ -4,9 +4,19 @@
...
+  <maintainer email="hamilton@mbari.org">Andrew Hamilton</maintainer>
+  <maintainer email="dharini@openrobotics.org">Dharini Dutia</maintainer>
+  <maintainer email="mabel@openrobotics.org">Mabel Zhang</maintainer>
+  <maintainer email="anderson@mbari.org">Michael Anderson</maintainer>
+
   <license>Apache 2.0</license>
+
   <author>Louise Poubel</author>
+  <author>Joan Aguilar Mayans</author>
+  <author email="michael@openrobotics.org">Michael Caroll</author>
```
Comment (mabelzhang) [inline buoy_gazebo/package.xml:17]:
```suggestion
  <author email="michael@openrobotics.org">Michael Carroll</author>
```

File: buoy_tests/package.xml [hunk 3]
```diff
@@ -4,13 +4,18 @@
...
-  <maintainer email="louise@openrobotics.org">Louise Poubel</maintainer>
+
+  <maintainer email="hamilton@mbari.org">Andrew Hamilton</maintainer>
+  <maintainer email="anderson@mbari.org">Michael Anderson</maintainer>
+
   <license>Apache 2.0</license>
+
+  <author email="dharini@openrobotics.org">Dharini Dutia</author>
   <author>Louise Poubel</author>
+  <author email="michael@openrobotics.org">Michael Caroll</author>
```
Comment (mabelzhang) [inline buoy_tests/package.xml:15]:
```suggestion
  <author email="michael@openrobotics.org">Michael Carroll</author>
```

**Review by mabelzhang — COMMENTED** [review]
I think it's fine to have the author tags without emails. Since they're not maintainers, people shouldn't be bothering them anyway.

**Review by mabelzhang — APPROVED** [review]
Looks good to me.
Feel free to merge unless there's dissent - can always correct later.

**Comment by quarkytale** [comment]
Not sure why tests are failing, cannot reproduce locally.

File: .github/workflows/build-and-test.sh [hunk 4]
```diff
@@ -34,10 +34,6 @@ cp -r $GITHUB_WORKSPACE $COLCON_WS_SRC
 wget https://raw.githubusercontent.com/osrf/mbari_wec/main/mbari_wec_all.yaml
 vcs import --skip-existing < mbari_wec_all.yaml
 
-rosdep init
-rosdep update
-rosdep install --from-paths ./ -i -y -r --rosdistro $ROS_DISTRO
-
 git clone https://github.com/hamilton8415/FreeSurfaceHydrodynamics.git
```
Comment (andermi) [inline .github/workflows/build-and-test.sh:37]: @hamilton8415 should CI switch to using the ppa?

Comment (quarkytale) [reply .github/workflows/build-and-test.sh:37]: Same question, ppa works locally

Comment (quarkytale) [reply .github/workflows/build-and-test.sh:37]: Actually no, I'm able to reproduce the `apt: Failed to detect successful installation of [ros-humble-actuator-msgs]` using the Dockerfile with ppa install of FreeSurfaceHydrodynamics

**Comment by quarkytale** [comment]
SC Commands test keeps timing out here which seems unrelated to the changes in this PR. @andermi any suggestions?
` Test #37: launch_sc_commands_ros_feedback.launch.py ...***Timeout 900.11 sec`

**Comment by andermi** [comment]
Increase the timeouts. It’s a long test on a very slow machine. I’ve had to increase the timeouts a few times now. I think their machines are getting even slower…

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by quarkytale** [comment]
@andermi set the timeouts to nothing less than 1000 sec works well, is it okay if I get this in and further refinements can be done in a separate PR if required?

**Comment by andermi** [comment]
Yep fine with me! Merge away!

**Pull request merged**